### DISCUSSION TITLE
Centralized/Decentralized

### DISCUSSION THREAD

**Discussion by ahmadkhan345** [wevm/wagmi#1938] [Q&A]
If I am making an application with wagmi hooks, and interacting with my metamask and etherscan, would it be considered a centralized or decentralized app?

**Comment by tmm** [comment]
It depends. What is your definition of centralized and decentralized? Some folks might consider what you describe to be centralized, decentralized, or somewhere in-between. I tend to view this as more of a spectrum versus binary.
